src/Microsoft.Azure.WebJobs.Host/WebJobs.Host.csproj

```diff
@@ -54,6 +54,14 @@
     <AssemblyOriginatorKeyFile>..\Common\PublicKey.snk</AssemblyOriginatorKeyFile>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="Microsoft.AI.PerfCounterCollector, Version=2.2.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+      <HintPath>..\..\packages\Microsoft.ApplicationInsights.PerfCounterCollector.2.2.0\lib\net45\Microsoft.AI.PerfCounterCollector.dll</HintPath>
```

Comment:
This won't work for Web Apps though, will it? If someone is using WebJobs, which runs as a console app, it'll work, but it wouldn't get hit when we run this in the context of a Web app like we do in Functions. I'm going to punt this -- we can probably come up with a solution to log all critical crashing failures in a follow-up.